hw/ip_templates/racl_ctrl/README.md

```diff
@@ -1 +1,119 @@
 # RACL Control Permission IP
+
+`racl_ctrl` is currently being used in [top_darjeeling.hjson](../../../../hw/top_darjeeling/data/top_darjeeling.hjson).
+There `racl_config` points to the main configuration file of the RACL Controller.
+E.g., `racl_config: 'top_darjeeling/data/racl/racl.hjson'`.
+
+Furthermore, each module instance that supports RACL must specify a valid `racl_mapping` for each of its `bus_interfaces`.
+If there is only one such interface, a racl mapping may be specified as follows:
+
```

Comment:
Nit: Is there a reason to include the blank lines like this? I think that some tools might interpret this as a paragraph break, which we don't need around the code fragments. (Obviously, it would depend on the word processor or whatever, but we don't want "whereas" to be indented)